deepchecks/nlp/utils/text_properties.py

```diff
@@ -35,6 +37,41 @@
 MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.nlp-models'
 FASTTEXT_LANG_MODEL = 'https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin'
 
+text_blob_cache = {}
+
+
+def _sample_for_property(text: str, mode: str = 'words', limit: int = 10000, random_seed: int = 42):
+    """Get a sample a single text sample for a text property.
+
+    Parameters
+    ----------
+    text : str
+        The text to sample from.
+    mode : str, default 'words'
+        The mode to sample in. Can be either 'characters', 'words' or 'sentences'.
+    limit : int, default 1000
+        The maximum number of words or sentences to sample.
+    """
+    np.random.seed(random_seed)
+    if mode == 'characters':
+        if len(text) <= limit:
+            return text
+        else:
+            all_units = np.random.choice(list(text), size=limit, replace=False)
```

Comment:
Dunno in what case you'll use that, but I would imagine that you'd want to sample in order

Reply:
Thought in Special Characters, actually just removed later as it didn't work for it